Makefile

```diff
@@ -361,7 +362,8 @@ setup: venv
 # TODO: for now, we only extract assets from the Debug ROM
 ifeq ($(VERSION),gc-eu-mq-dbg)
 	$(PYTHON) extract_assets.py -j$(N_THREADS)
-	$(PYTHON) tools/msgdis.py --text-out assets/text/message_data.h --staff-text-out assets/text/message_data_staff.h
+	mkdir -p $(EXTRACTED_DIR)/text
```

Comment:
should this go with the `create build directories` step above instead of here? Ditto with the one for `$(BUILD_DIR)` just below

Reply:
May be personal preference (out of what I believe to be good practice) but I prefer running commands as part of rules (as intended) rather than as part of the Makefile being processed and parsed

Reply:
I'm not dying on this hill though I can also merge it into the global mkdir

Reply:
Yeah I agree running `mkdir` when eval-ing the makefile is weird and running it as part of a rule is better. But I think it's even better to be consistent now and change them all at once (e.g. https://www.twoistoomany.com/blog/2020/07/23/make-order-only-prerequisites/ claims that order-only prerequisites would be even better than a mkdir in each rule, but I haven't looked into it)